python/src/robyn/modeling/transformations/transformations.py

```diff
@@ -0,0 +1,136 @@
+# pyre-strict
+
+import numpy as np
+import pandas as pd
+from typing import List, Dict, Any, Tuple, Optional
+
+from robyn.data.entities.enums import AdstockType
+
+
+class Transformations:
+    @staticmethod
+    def mic_men(x: np.ndarray, Vmax: float, Km: float, reverse: bool = False) -> np.ndarray:
```

Comment:
1. x is input data that will be common for most methods. Let's make that instance variable
2. Use meaningful variable name. "x" isn't meaningful variable name,